README.rst

```diff
@@ -129,16 +249,16 @@ Executing a query
     results = cursor.fetchall()
```

Comment:
This is a python spanner example of running a query, we don't want that for django. Please show how to insert data in django and how to query it.